syserol/tensor.py

```diff
@@ -191,16 +234,23 @@ def perform_CMTF(tOrig=None, r=6):
     # Precalculate the missingness patterns
     uniqueInfo = [np.unique(np.isfinite(B.T), axis=1, return_inverse=True) for B in unfolded]
 
+    # get unique days into vector format for continuous solve
+    days = dayLabels()
+    # initialize parameter matrix
+
     for ii in range(2000):
         # PARAFAC on all modes
         for m in range(0, len(tFac.factors)):
             kr = khatri_rao(tFac.factors, skip_matrix=m)
             tFac.factors[m] = censored_lstsq(kr, unfolded[m].T, uniqueInfo[m])
         
+        # for final (continuous) dimension, solve for P and recalculate continuous factor
+        tFac.factors[m] = continue_solve(days, tFac.factors[m])
```

Comment:
How does `continue_solve()` work if it doesn't have the unfolded data?

Reply:
Oh, I see now you're using a simpler strategy in the mean time. This is fine.

Reply:
Continue solve does these steps:
2. Using our known A_hat, reverse solve for the parameter matrix P that best fits A_hat to a sigmoidal function given time vector v.
3. With newly solved P, plug P and v into our sigmoidal function, and output new A_hat